modules/configuring-hybrid-ovnkubernetes.adoc

```diff
@@ -18,6 +18,22 @@ You must configure hybrid networking with OVN-Kubernetes during the installation
 
 * You defined `OVNKubernetes` for the `networking.networkType` parameter in the `install-config.yaml` file.
 
+You can do this by first creating your `install-config.yaml` fine for your cluster.
+
+[source,terminal]
+----
+$ ./openshift-install create install-config --dir=<installation_directory>
+----
+
+Then, edit the `install-config.yaml` file to use `OVNKubernetes` as the `networkType`.
+
+[source,terminal]
+----
+$  sed -i 's/OpenShiftSDN/OVNKubernetes/g' installation_directory>/install-config.yaml
+----
+
+You are now ready to begin the procedure.
+
```

Comment:
There is already a prereq for setting `OVNKubernetes` for the `networking.networkType` parameter in the install config in this doc. The reason this was broken out like this is because this module is also included in the network installation sections for [AWS](https://docs.openshift.com/container-platform/4.6/installing/installing_aws/installing-aws-network-customizations.html#configuring-hybrid-ovnkubernetes_installing-aws-network-customizations) and [Azure](https://docs.openshift.com/container-platform/4.6/installing/installing_azure/installing-azure-network-customizations.html#configuring-hybrid-ovnkubernetes_installing-azure-network-customizations). For consistency, this follows [Modifying advanced network configuration parameters
](https://docs.openshift.com/container-platform/4.6/installing/installing_azure/installing-azure-network-customizations.html#modifying-nwoperator-config-startup_installing-azure-network-customizations).

We could change the current link in [Enabling WINC](https://docs.openshift.com/container-platform/4.6/windows_containers/enabling-windows-container-workloads.html) to point to those assemblies instead if you think that makes more sense. I was avoiding this, though, because they are cloud provider-specific, and will be a pain to maintain as more cloud providers are supported with WINC.

Reply:
ACK.

The reason I was thinking of adding this was because I was trying to test something and was following the docs. It wasn't working then I remembered "oh I forgot to edit the install-config.yaml" file.  Admittedly, this is a case of I didn't RTFM :)

This is easy now because it's AWS/Azure.

But thinking about doing BYO Windows host on a Platform Agnostic UPI (in the future), "Just reinstall" is basically burning a half  day.